app/views/kaui/admin_tenants/_show_catalog_simple.erb

```diff
@@ -43,10 +43,10 @@
       <th></th>
     </tr>
     </thead>
-    <tbody>
+    <tbody id="catalog_detail">
     <% @catalogs.each do |catalog| %>
         <% catalog[:plans].each do |p| %>
-            <tr class="catalog_version_<%= catalog[:version] %>">
+            <tr class="selected_catalog">
```

Comment:
Out of curiosity, would it be possible to remove completely this and use the template instead on page load? That way, the template wouldn't be duplicated. If it's too hard/cumbersome/... forget about it.